addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -99,6 +99,7 @@ <H2>Binary Incompatible Changes</H2>
 <li><code>org.zaproxy.zap.spider.SpiderParam#getScopeText()</code></li>
 <li><code>org.zaproxy.zap.view.ContextExcludePanel#getPanelName(Context)</code></li>
 <li><code>org.zaproxy.zap.view.ContextIncludePanel#getPanelName(Context)</code></li>
+  <li><code>org.parosproxy.paros.control.Control#createAndOpenUntitledDb()</code></li>
```

Comment:
This should be in alphabetical order (so higher up) and not indended.
But other than that its good :)